packages/ethereum-contracts/package.json

```diff
@@ -20,7 +20,8 @@
         "/build/contracts-sizes.txt",
```

Comment:
Will we increase the version in another pr?

Reply:
yeah, I think once we want to do a release it will make sense to - once we complete the cfav1 forwarder changes + maybe: https://github.com/superfluid-finance/protocol-monorepo/issues/1112